scripts/run-dependencies.sh

```diff
@@ -51,3 +51,4 @@ docker run --rm --network=$NETWORK \
   -url=jdbc:postgresql://$DB/hint
 
 $HERE/add-test-user.sh
+$HERE/add-support-user.sh
```

Comment:
Without this testing out the issue report on local deployment is raising an error as support user doesn't exist